docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_RetinaNet_From_Tensorflow.md

```diff
@@ -0,0 +1,12 @@
+# Converting RetinaNet Model from TensorFlow* to the Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_tf_specific_Convert_RetinaNet_From_Tensorflow}
+
+This tutorial explains how to convert RetinaNet model to Intermediate Representation (IR).
+
+[Public RetinaNet model](https://github.com/fizyr/keras-retinanet) does not contain pretrained TensorFlow* weights. To convert this model to the TensorFlow* format you can use [Reproduce Keras* to TensorFlow* Conversion tutorial](https://docs.openvinotoolkit.org/latest/omz_models_model_retinanet_tf.html).
+
+After you convert model to TensorFlow* format you can run the Model-Optimizer command below:
+```sh
+python mo.py --input "input_1[1 1333 1333 3]" --input_model retinanet_resnet50_coco_best_v2.1.0.pb --data_type FP32 --transformations_config ./extensions/front/tf/retinanet.json
+```
+
+Where `transformations_config` command line parameter specifies the configuration json file containing hints to the Model Optimizer on how to convert the model. The json file contains some parameters which must be changed if you train the model yourself. Also it contain how to matching end pints for replacing subgraph nodes. After converting model to IR output nodes will be replaced to DetectionOutput layer.
```

Comment:
please split these sentences into several lines. Otherwise, it is difficult to maintain it in sources

Reply:
contain->contains